pandas/tests/indexes/multi/test_set_ops.py

```diff
@@ -9,91 +9,110 @@
 
 
 @pytest.mark.parametrize("case", [0.5, "xxx"])
+@pytest.mark.parametrize("sort", [True, False])
 @pytest.mark.parametrize("method", ["intersection", "union",
                                     "difference", "symmetric_difference"])
-def test_set_ops_error_cases(idx, case, method):
+def test_set_ops_error_cases(idx, case, sort, method):
     # non-iterable input
     msg = "Input must be Index or array-like"
     with pytest.raises(TypeError, match=msg):
-        getattr(idx, method)(case)
+        getattr(idx, method)(case, sort=sort)
 
 
-def test_intersection_base(idx):
+@pytest.mark.parametrize("sort", [True, False])
+def test_intersection_base(idx, sort):
     first = idx[:5]
     second = idx[:3]
-    intersect = first.intersection(second)
+    intersect = first.intersection(second, sort=sort)
 
+    if sort:
+        tm.assert_index_equal(intersect, second.sort_values())
```

Comment:
yeah ideally we would be able to remove tm.equalContents, just in favor or assert_almost_equal; the writer of the test can have a deterministic ordering (rather than leave it up to a set impl).